gallery/src/styles/NxCard/NxCardVerticalExample.tsx

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { faShapes, faChartLine, faEdit, faTrash } from '@fortawesome/free-solid-svg-icons';
+
+import { NxFontAwesomeIcon, NxButton } from '@sonatype/react-shared-components';
+
+export default function NxCardVerticalExample() {
+  return (
+    <>
+      <div className="nx-card-container nx-card-container--row">
+        <section className="nx-card">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">Header</h3>
+          </header>
+          <div className="nx-card__call-out">
+            42
+          </div>
+          <div className="nx-card__content">
+            Text
+          </div>
+          <footer className="nx-card__footer">
+            <a href="#" className="nx-text-link">Link</a>
+          </footer>
+        </section>
+        <section className="nx-card">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">Card header</h3>
+          </header>
+          <div className="nx-card__call-out">
+            <NxFontAwesomeIcon icon={faShapes} className="nx-icon--card-call-out" />
+          </div>
+          <div className="nx-card__content">
+            Data point details
+          </div>
+          <footer className="nx-card__footer">
+            <a href="#" className="nx-text-link">Link</a>
+          </footer>
+        </section>
+        <section className="nx-card">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">Card header</h3>
+          </header>
+          <div className="nx-card__call-out">
+            <NxFontAwesomeIcon icon={faShapes} className="nx-icon--card-call-out-xl" />
```

Comment:
Do these `.nx-icon` modifiers need to be documented?

Reply:
> Do these `.nx-icon` modifiers need to be documented?

I waffled on that and in the end documented them here: https://github.com/sonatype/sonatype-react-shared-components/pull/170/commits/29682dcab30f0d9ca6ffd4b490f5764de425794e

Though I still think that we should consider making some basic icon sizing classes for common values.